### PR TITLE
webui: Add inline alert to the Storage screen

### PR DESCRIPTION
**Pull request by M4rtinK**

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Comment by M4rtinK** [comment]
no_alert
So this is how it used to look without the alert.
alert
This is how it looks now.

File: ui/webui/src/components/storage/InstallationDestination.jsx [hunk 1]
```diff
@@ -213,6 +214,15 @@ export const InstallationDestination = ({ idPrefix, onAddErrorNotification }) =>
             <HelperText>
                 <HelperTextItem>{_("Select the device(s) you would like to install to")}</HelperTextItem>
             </HelperText>
+            <Alert
+              isInline
+              variant="info"
+              title={_("Selected disks will be erased at at install")}
```
Comment (poncovka) [inline ui/webui/src/components/storage/InstallationDestination.jsx:220]: at at

**Review by poncovka — CHANGES_REQUESTED** [review]

**Comment by M4rtinK** [comment]
After rebasing on latest master branch, it looks like there is a subtle glitch where the icon is not aligned with the text:
TestStorage-testLocalStandardDisks-storage-step-basic-pixels

As the code in the PR worked before yet is still the same, this is likely a non-fatal side effect of one of the recent PRs and can be fixed later.

File: ui/webui/src/components/storage/InstallationDestination.jsx (hunk 1 above)
Comment (poncovka) [reply ui/webui/src/components/storage/InstallationDestination.jsx:282]: Well, it was merged with the typo.

**Pull request merged**